client/editSchedule.js

```diff
@@ -190,46 +158,31 @@ Template.addEditSomethingModal.helpers({
 });
 
 AutoForm.hooks({
-  editRoundForm: {
+  editEventForm: {
     onSubmit: function(insertDoc, updateDoc, currentDoc) {
       this.event.preventDefault();
 
       if(currentDoc._id) {
-        // Updating an existing round
-        Rounds.update({
-          _id: currentDoc._id,
-        }, {
-          $set: {
-            title: insertDoc.title,
-            startMinutes: insertDoc.startMinutes,
-            durationMinutes: insertDoc.durationMinutes,
-          }
-        });
-        $('#addEditSomethingModal').modal('hide');
+        delete updateDoc.$set.competitionId; // Don't touch the competitionId field. We don't trust browser input that much.
```

Comment:
Ohhh, I see. The old code specifically whitelisted fields to send over, whereas this new code blacklists.

LOL at the TSA analogy, I like it =)